dhis-2/dhis-api/src/main/java/org/hisp/dhis/datavalue/DataValueAudit.java

```diff
@@ -116,6 +116,22 @@ public static DataValueAudit from( DataValueAudit dataValueAudit )
             dataValueAudit.getAuditType() );
     }
 
+    public static DataValueAudit from( DataValueAudit dataValueAudit, Date created )
```

Comment:
Feels like `@Builder` with `toBuilder()` might be a more concise way doing this.